atlas-db-controller/databaseSchemaResource.go

```diff
@@ -15,6 +17,11 @@ import (
 
 var dbDriverMap = make(map[string]database.Driver)
```

Comment:
Just a thought, putting it out there. 
Will the below code be more clear and readable ? 
```go
type DBCache map[string]database.Driver

func (cache *DBCache) Add(key, value) {

}

func (cache *DBCache) Delete(key) {
}

```

Reply:
Sounds like a good idea. Although needs extensive testing with `Add`, `Delete` and `New`. Will take it up later.